.github/workflows/release.yml

```
@@ -47,7 +47,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
```

Mind changing this to use:

```suggestion
          go-version-file: 'path/to/go.mod'
```

https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file